database/config.go

```diff
@@ -158,20 +158,23 @@ func (t *tableInfoStore) Insert(tableName string, info *TableInfo) ([]byte, erro
 	}
 
 	var id [6]byte
-	for {
-		id = generateStoreID()
-		_, err = t.st.Get(id[:])
-		if err == nil {
-			// A store with this id already exists.
-			// Let's generate a new one.
-			continue
+	// Generate a new storeID if the one in the info is empty.
+	if info.storeID == id {
```

Comment:
To avoid this logic and since it's only useful the first time we create a table, maybe we should generate the store id outside of this function, in the `Insert` method.

Reply:
Done